File-Formats/DocIO/Getting-Started.md

```diff
@@ -1807,4 +1807,10 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Result.pdf", "applicat
 
 N> 
 When the ChartToImageConverter object is not initialized, then the charts in Word document gets skipped during Word to PDF conversion.
-ChartToImageConverter is supported from .NET Framework 4.0 onwards
\ No newline at end of file
+ChartToImageConverter is supported from .NET Framework 4.0 onwards
+
+## See Also
+
+* [How to generate Word document in C#, VB.NET](https://www.syncfusion.com/kb/12989/how-to-generate-word-document-in-c-vb-net)
```

Comment:
Add content like

Create Word document in ASP.NET Core -- redirect to UG links
Create Word document in Xamarin -- redirect to UG links
Similarly, add all platform UG pages links